minitask/new_import_data_v2.py

```diff
@@ -0,0 +1,162 @@
+
+import pandas as pd
+import requests
+import json
+import gzip
+from lxml import etree
+
+import wikipedia, gzip, threading
+from urllib.request import urlopen
+from elasticsearch import Elasticsearch as es
+from wikipedia.exceptions import PageError, DisambiguationError
+import ssl
+from multiprocessing import Pool
+import multiprocessing
+
+from elasticsearch import Elasticsearch as es
+
+from elasticsearch import helpers as h
+
+from dataclasses import dataclass
+
+@dataclass
+class Abstract:
+    """Wikipedia abstract"""
+    ID: int
+    title: str
+    abstract: str
+    url: str
+
+    @property
+    def fulltext(self):
+        return ' '.join([self.title, self.abstract])
+
+"""
+Need to create the index first before using this script
+curl -XPUT https://localhost:9200/example3 --insecure -u admin:admin
+"""
+URL = 'https://localhost:9200/example3/_doc/'
+
+# the code with idea inspired by https://www.cnblogs.com/shaosks/p/7592229.html
+
+def start_import(test_size = 1000):
```

Comment:
**[Required]** We shouldn't be using test_sizes by default. All references to this should be removed.